core/modules/wsched/ChunkTaskList.cc

```diff
@@ -0,0 +1,387 @@
+// -*- LSST-C++ -*-
```

Comment:
It would be nice to have a detailed documentation with a few schema once this code will be stable, maybe in a future ticket?